internal/health_checks/network_check.go

```diff
@@ -0,0 +1,71 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package health_checks
+
+import (
+	"log"
+	"net"
+	"net/http"
+)
+
+var (
+	// API urls
+	loggingAPIUrl    = "https://logging.googleapis.com/$discovery/rest"
+	monitoringAPIUrl = "https://monitoring.googleapis.com/$discovery/rest"
+)
+
+type NetworkCheck struct{}
+
+func (c NetworkCheck) Name() string {
+	return "Network Check"
+}
+
+func (c NetworkCheck) RunCheck(logger *log.Logger) error {
+
```

Comment:
```suggestion
```

Reply:
Addressed in https://github.com/GoogleCloudPlatform/ops-agent/pull/1051/commits/6b5c1d6337aa4b35abe00f713626d12a0658df7a .